### ISSUE TITLE
CC Thread breaks on world reload

### ISSUE THREAD

**Issue by hugeblank** [mystiacraft/cc-tweaked-fabric#16]
- MC 1.14.4
 - CCTF: 1.83.2 build.7
 - Reproduction steps:
1. Load a SP world
2. Save & Quit, do not exit game
3. Log back in

Expected Result:
CC would start computers up, startup scripts would get executed, etc. 

Actual Result:
All computers render a black screen, appearing to be completely locked up.

**Comment by parly** [comment]
Hmm, I can't reproduce your problem. In my environment, startup scripts are working correctly after logging back in (tested in a new world).
Are there any other conditions for reproducing the situation, such as running a specific script?

**Comment by hugeblank** [comment]
https://github.com/mystiacraft/cc-tweaked-fabric/commit/9967dc57408a888f6f5da6ca327536d52db64228 May have fixed this issue if it was just a matter of shading Cobalt. I was also having an issue with my original build file not starting the CC Thread, but instead of on world reload it was right on start up. I’ll test this and come back to you with more info

**Comment by hugeblank** [comment]
Unlike what I thought in my last message, fixing the buildfile (which, on a positive note looks _much_ cleaner) didn't resolve the issue of the CC thread not starting in production.

**Comment by parly** [comment]
So I haven't reproduced this issue yet. In my env, it works fine... 🤔
I'll continue to investigate, but it would be helpful if you tell me when you find out something about this.

**Comment by hugeblank** [comment]
Have you tried using a production copy of the mod? That’s where this issue occurs.

**Comment by parly** [comment]
Yes, I've tested using build.8 on CurseForge. Today I newly tested on a clean Windows 10 VM, but it worked...

If your development version works for you, can you give me the .jar file, please? I'd like to disassemble both copies and compare what the difference is.

**Comment by hugeblank** [comment]
Well when I build a production copy myself from the development environment that works, it incurs the same issues as the one on CF. [Here](https://drive.google.com/file/d/1B9i6P6be_umYuB-eLlsi99JCooseHhop/view?usp=sharing) it is anyways, you most likely have a better idea of what's going on than me.

**Comment by parly** [comment]
That's bad. I looked into both copies just in case, but there were no differences that seemed to be related to this issue. (But thanks to your .jar file, other potential problems when building with Java 9+ were found and [fixed](https://github.com/mystiacraft/cc-tweaked-fabric/commit/429baa350ca9de33b3551bdff8264b8c92b2ce6a)!)

Could it be worked by resetting the configuration by deleting `.minecraft/config/computercraft.json5`, and restarting Minecraft? To be honest, I can't come up with any other solutions anymore...

**Comment by hugeblank** [comment]
yeah resetting the config didn't fix anything. I'm getting a friend to test this issue on Windows to see if maybe it's a mac inconsistency.

**Comment by hugeblank** [comment]
okay... apparently this is just a MacOS issue. My friend says that everything works fine. idk if you want to consider this an edge case issue and close it or what because it's so odd, and it's not that big of a deal.
